### PR TITLE
feat: add launchpad downloads

### PR DESCRIPTION
**Pull request by NovaT82**
Added new images / videos for the launchpad downloads section.
Cleaned up the downloads layout on the homepage.
Put in a ternary statement to check on which page the `downloads.js` file is being called and depending on the location a different url will be called for the downloads fetch. 
Awaiting the final url for the launchpad section - this will need to be updated for the Launchpad downloads section to work.

### CODE REVIEW

File: _data/launchpad_downloads.yml [hunk 1]
```diff
@@ -22,7 +22,7 @@
   archOptions:
     [{ value: arm64, label: arm64 }, { value: x86_64, label: x86_64 }]
   networkOptions:
-    [{ value: nextnet, label: Nextnet }, { value: stagenet, label: Stagenet }]
+    [{ value: nextnet, label: Nextnet }, { value: testnet, label: Testnet }]
```
Comment (CjS77) [inline _data/launchpad_downloads.yml]: There is no "testnet"

It's either stagenet or nextnet

Comment (NovaT82) [reply _data/launchpad_downloads.yml]: I updated it to only use nextnet

File: _includes/head.html [hunk 2]
```diff
@@ -99,7 +99,8 @@
           dataKey: "tari-project/wallet-ios",
           elementPrefix: "wallet",
         }
-      ]
+      ],
+      networks: ["nextnet", "testnet"],
```
Comment (CjS77) [inline _includes/head.html]: There's no `testnet`

Comment (NovaT82) [reply _includes/head.html]: Sorry, I missed that one - removed it

**Review by CjS77 — APPROVED** [review]

**Pull request merged**